hack/arktos-apiserver-partition.md

```diff
@@ -0,0 +1,170 @@
+
+# Arktos Apiserver Partition
+
+[![Go Report Card](https://goreportcard.com/badge/github.com/futurewei-cloud/arktos)](https://goreportcard.com/report/github.com/futurewei-cloud/arktos)
+[![LICENSE](https://img.shields.io/badge/license-apache%202.0-green)](https://github.com/futurewei-cloud/arktos/blob/master/LICENSE)
+
+
+## What Arktos Apiserver Partition is
+Arktos Apiserver Partition is a lightweight componizing script to deploy Arktos by its function declarations. It runs like bash hack/arktos-apiserver-partition.sh function_name to achieve flexiblities.
+
+## Key Features of Arktos Apiserver Partiion
+
+### Partitioned 
+Feel free to export RANGE_BEGIN=aaa, RANGE_END=zzz to make apiservers only watches data you want to synchronize
+
+### Deploy across hosts
+Since we build a shared ETCD cluster, all the data can be shared across hosts
+
+### Build Arktos
+
+
+To build Arktos, you just need to clone the repo and run "make". There is a git take missing issue to build images. It can be fixed by running "git tag -a v2.7.4"
+
+##### Note: you need to have a working [Go 1.12 environment](https://golang.org/doc/install). Go 1.13 is not supported yet.
+
+```
+mkdir -p $GOPATH/src/github.com
+cd $GOPATH/src/github.com
+git clone https://github.com/futurewei-cloud/arktos
+```
+
+Please make it sure run make clean if you have any changes to apply
+
+
+## Deployment Steps
+
+
+To run at multiple hosts
+
+Install ETCD
+
+```
+cd $GOPATH/src/github.com/arktos
+bash hack/install_etcd.sh install_etcd
+```
+
+Run ETCD
+
+```
+bash hack/install_etcd.sh start_etcd
```

Comment:
This does not work. Also do you need to add instruction for "export PATH="/home/ubuntu/go/src/arktos/third_party/etcd:${PATH}"?

Reply:
export does not work in bash